src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -3146,3 +3160,58 @@ def get_esol(
         )
         # Using get_variable because it deletes the intermediate parameter after using it.
         return self.get_variable(VAR_IR, tstrt=tstrt, kcplx=kcplx)
+
+    def _create_session(self):
+        """Generate a session ID."""
+        id_ = uuid4()
```

Comment:
@germa89 local variables shouldn't have a trailing underscore

Reply:
> This is necessary because id is a built-in.

Exactly that. :)